modules/migration-about-migrating-applications-api.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration-toolkit-for-containers/advanced-migration-options-mtc.adoc
+
+[id="migration-about-migrating-applications-api_{context}"]
+= About migrating applications
+
+You can migrate applications from a local cluster to a remote cluster, from a remote cluster to a local cluster, and between remote clusters.
+
+[id="terminology_{context}"]
+== Terminology
+
+Source cluster::
+* Cluster from which the applications are migrated.
+
+Destination cluster::
+* Cluster to which the applications are migrated.
+
+Replication repository::
+* Object storage.
+* Requires network access to all clusters.
+
+Indirect migration::
+* Images, volumes, and Kubernetes objects are copied from the source cluster to the replication repository and then from the replication repository to the destination cluster.
+
+Direct volume migration::
+* Volumes are copied directly from the source cluster to the destination cluster.
+* Significantly faster than indirect migration.
+
+Direct image migration::
+* Images are copied directly from the source cluster to the destination cluster.
+* Significantly faster than indirect migration.
+
+Host cluster::
+* Cluster on which the `migration-controller` pod and the web console run.
+* Usually the same as the destination cluster and the local cluster but this is not a requirement.
+* Does not require an exposed secure registry route for direct image migration.
+
+Local cluster::
+* Usually the same as the destination cluster and the host cluster but this is not a requirement.
+
+Remote cluster::
+* Usually the same as the source cluster but this is not a requirement.
+* Requires an exposed secure registry route for direct image migration.
+* Requires a `Secret` CR containing the `migration-controller` service account token.
+
+[id="mapping-destination-namespaces-in-the-migplan-custom-resource_{context}"]
+== Mapping destination namespaces in the MigPlan custom resource (CR)
+
+If you map destination namespaces in the `MigPlan` CR, you must ensure that the namespaces are not duplicated on the source or the destination clusters because the UID and GID ranges of the namespaces are copied during migration.
+
+The following examples will cause validation errors:
+
+.Two source namespaces mapped to the same destination namespace
+[source,yaml]
+----
+spec:
+  namespaces:
+    - namespace_2
+    - namespace_1:namespace_2
+----
+
+.Source namespace mapped to duplicate namespace
+[source,yaml]
+----
+spec:
+  namespaces:
+    - namespace_1:namespace_1
```

Comment:
The above isn't actually an error, although it is redundant. "namespace_1:namespace_1" is equivalent to "namespace_1".

Reply:
It  will report error if it's "namespace_1:namespace_1" on UI

Reply:
@eriknelson We may want to flag this as a UI issue to investigate (and also confirm that it does actually work on the back end), but I think it's confusing to call this a "duplicate namespace" issue since this is no more duplication than a normal non-mapped scenario. Maybe the UI guide should simply say that if source and dest namespace are the same, then the namespace should only be named once here.